src/main/java/racingCar/Car.java

```diff
@@ -1,24 +1,50 @@
 package racingCar;
 
 public class Car {
-    private int distance;
     private static final int LIMIT_NUM = 4;
+    private static final int LIMIT_NAME_LENGH = 5;
+    private int distance;
+    private String name;
 
-    public Car() {
+    public Car(String name) {
+        checkName(name);
+        this.name = name;
         this.distance = 0;
```

Comment:
```suggestion
        this(name, 0);
```
이와 같이 주 생성자를 호출해 구현할 것을 추천
중복 코드를 제거하는 측면에서도 확실히 유리함